sell.php

```diff
@@ -92,7 +92,7 @@
 			$phpass = new PasswordHash(8, false);
 			if (!($phpass->CheckPassword($_POST['password'], $user->user_data['password'])))
 			{
-				$ERR = 'ERR_026';
+				$ERR = $ERR_006;
```

Comment:
This might break this page, as dataCheck returns a string like 'ERR_xxx' that needs to be changed too